user-guide/Advanced_Functionality/Failover/Configuring_Failover/Failover_configuration_in_Cube.md

```diff
@@ -9,6 +9,9 @@ uid: Failover_configuration_in_Cube
 
 ## [From DataMiner 10.1.8 onwards](#tab/tabid-1)
 
+> [!NOTE]
+> Failover is automatically configured as part of [DaaS](xref:Creating_a_DMS_in_the_cloud). From DataMiner 10.3.0 [CU18]/10.4.0 [CU6]/10.4.9 onwards<!--RN 40013-->, the *Failover* button is therefore no longer available on the *Apps* > *System Center* > *Agents* page if you are using a DaaS system.
```

Comment:
It is not auto configured. It is disabled so we don't promote failover anymore. For example: we don't want users want to create failover pair with DaaS.
this is also soft-promoting of the Swarming feature.

Reply:
```suggestion
> If you are using a [DaaS system](xref:Creating_a_DMS_in_the_cloud), your DMS is fully hosted and maintained by Skyline Communications, which means you do not need to set up Failover. As such, from DataMiner 10.3.0 [CU18]/10.4.0 [CU6]/10.4.9 onwards<!--RN 40013-->, the *Failover* button is therefore no longer available on the *Apps* > *System Center* > *Agents* page if you are using a DaaS system.
```

Is this better, @HansSolos?

Reply:
```suggestion
> If you are using a [DaaS system](xref:Creating_a_DMS_in_the_cloud), your DMS is fully hosted and maintained by Skyline Communications. Therefore, a Failover configuration is not available for DaaS<!--RN 40013-->.
```